freebasics/models.py

```diff
@@ -0,0 +1,18 @@
+from wagtail.contrib.settings.models import BaseSetting, register_setting
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from django.db import models
+
+
+@register_setting
+class HomePageSettings(BaseSetting):
```

Comment:
I think we should call this `SiteSettings` because we might need to add other settings here